test/integration/organizations/gems_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+class Organizations::GemsTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = create(:user, remember_token_expires_at: Gemcutter::REMEMBER_FOR.from_now)
+    post session_path(session: { who: @user.handle, password: PasswordHelpers::SECURE_TEST_PASSWORD })
+  end
+
+  test "should get index" do
+    @organization = create(:organization, owners: [@user])
+    @organization.rubygems << create(:rubygem, name: "arrakis", number: "1.0.0")
+
+    get "/organizations/#{@organization.to_param}/gems"
+
+    assert page.has_content? "arrakis"
```

Comment:
```suggestion
   assert_content "arrakis"
```

Reply:
I think that works in other testing contexts and not here :( I hope we can resolve this discrepancy soon.